Gems/ScriptCanvas/Code/Include/ScriptCanvas/AutoGen/ScriptCanvasNodeable_Header.jinja

```diff
@@ -26,6 +26,9 @@ SPDX-License-Identifier: Apache-2.0 OR MIT
 #include <ScriptCanvas/Core/Nodeable.h>
 #include <ScriptCanvas/Core/NodeableNode.h>
 
+#include <ScriptCanvas/AutoGen/ScriptCanvasAutoGenRegistry.h>
```

Comment:
Side Note: The include of the SerializeContext, BehaviorContext and EditContext can be avoided and replaced with a forward declaration of `AZ::ReflectContext` to save including the most expensive headers.

Reply:
I've made a note to come back to this after this change is merged